hr_timesheet_sheet_period/models/res_company.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+SHEET_RANGE_PAYROLL_PERIOD = 100
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    sheet_range = fields.Selection(
+        selection_add=[
+            (SHEET_RANGE_PAYROLL_PERIOD, "Payroll Period"),
```

Comment:
This causes an issue why builds fail, key needs to be a string https://github.com/OCA/timesheet/blob/14.0/hr_timesheet_sheet/models/res_company.py#L20

Reply:
@alexey-pelykh this was it thanks! i missed it completely